src/rngs/os.rs

```diff
@@ -660,8 +661,7 @@ mod imp {
 }
 
 #[cfg(all(target_arch = "wasm32",
-          not(target_os = "emscripten"),
-          not(feature = "stdweb")))]
+          not(any(target_os = "emscripten", feature = "stdweb"))))]
```

Comment:
We still have no reason why the stdweb implementation is even used for the emscripten target, which was already working perfectly fine.